pkg/crd/kubermatic/v1/datacenter.go

```diff
@@ -152,7 +152,7 @@ type DatacenterSpec struct {
 
 	//nolint:staticcheck
 	//lint:ignore SA5008 omitgenyaml is used by the example-yaml-generator
-	Fake *DatacenterSpecFake `json:"fake,omitempty,omitgenyaml"`
+	Fake *DatacenterSpecFake `json:"fake,omitempty,omitempty"`
```

Comment:
nit: looks like duplicated `omitempty` parameter, is that intentional?

Reply:
nope, this is totally unintentional. Thx for pointing out ;-)